src/data/battler-tags.ts

```diff
@@ -1,6 +1,6 @@
 import { CommonAnim, CommonBattleAnim } from "./battle-anims";
 import { CommonAnimPhase, MoveEffectPhase, MovePhase, PokemonHealPhase, ShowAbilityPhase, StatChangePhase } from "../phases";
-import { getPokemonMessage, getPokemonNameWithAffix } from "../messages";
+import { getPokemonNameWithAffix } from "../messages";
```

Comment:
```suggestion
import { getPokemonMessage, getPokemonNameWithAffix } from "../messages";
```
Someone add message which is not localized so it has to be leave. (getPokemonMessage)